src/game/models/equips.js

```diff
@@ -310,7 +310,7 @@ export default [
   {
     id: 'ice_ring' ,
     name: 'Ice Ring',
-    description: 'It looks as powerfull as it shines.',
+    description: 'Ice rink?! Imagine how cray, da &^@#ing szie fo dose s#!%!!!1!',
```

Comment:
The typos left (including this bit) are on purpose. Tried to keep it lightweight